tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/ServicePossibleValuesService.java

```diff
@@ -28,15 +30,29 @@
 
 public class ServicePossibleValuesService extends PossibleValuesService
 {
+
     @Override
     public Status problem( final Value<?> value )
     {
         return Status.createOkStatus();
     }
-   
+
     @Override
     protected void compute( final Set<String> values )
     {
-        values.addAll( Arrays.asList( NewLiferayModuleProjectOpMethods.getServices() ) );
+        try
```

Comment:
The server may not always be available.  This needs to be able to check if it can make a connection first and if so, then it can issue the command but if it can't make services connection then it should use the default list.